tests/tck/features/match/MatchById.feature

```diff
@@ -973,3 +973,68 @@ Feature: Match By Id
       | ("Warriors" :team{name: "Warriors"})                                                                        |
       | ("Hornets" :team{name: "Hornets"})                                                                          |
       | ("Spurs" :team{name: "Spurs"})                                                                              |
+
+  Scenario: match by multiple id
+    When executing query:
+      """
+      MATCH (a)-[e:like]->(b)
+      WHERE id(a) == 'Tim Duncan' OR id(b) == 'Tony Parker'
```

Comment:
This should report error

Reply:
use like index not vertexIDseek

Reply:
Emm. It's not related to this VIdSeek, better not put in this file.

Reply:
And you should add fail case of or expression.

Reply:
> Emm. It's not related to this VIdSeek, better not put in this file.

it's ok to put this case here